src/ProjectTemplates/Web.ProjectTemplates/content/Api-CSharp/Program.cs

```diff
@@ -0,0 +1,34 @@
+#if (NativeAot)
+using System.Text.Json.Serialization;
+#endif
+using Company.ApiApplication1;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Logging.AddConsole();
+
+#if (NativeAot)
+builder.Services.ConfigureHttpJsonOptions(options =>
+{
+    options.SerializerOptions.AddContext<AppJsonSerializerContext>();
+});
+
+#endif
+var app = builder.Build();
+
+var sampleTodos = TodoGenerator.GenerateTodos().ToArray();
+
+var todosApi = app.MapGroup("/todos");
+todosApi.MapGet("/", () => sampleTodos);
+todosApi.MapGet("/{id}", (int id) =>
+    sampleTodos.FirstOrDefault(a => a.Id == id) is { } todo
```

Comment:
Nit: `SingleOrDefault` seems more appropriate for resolving an element by a unique identifier.

Reply:
During a recent review of our API scaffolders we were advised that `SingleOrDefault` has extra overhead that isn't worth incurring in these scenarios so I adopted it here too.